pkg/cmd/server/server.go

```diff
@@ -153,6 +161,8 @@ func NewCommand() *cobra.Command {
 	command.Flags().StringSliceVar(&config.restoreResourcePriorities, "restore-resource-priorities", config.restoreResourcePriorities, "desired order of resource restores; any resource not in the list will be restored alphabetically after the prioritized resources")
 	command.Flags().StringVar(&config.defaultBackupLocation, "default-backup-storage-location", config.defaultBackupLocation, "name of the default backup storage location")
 	command.Flags().Var(&volumeSnapshotLocations, "default-volume-snapshot-locations", "list of unique volume providers and default volume snapshot location (provider1:location-01,provider2:location-02,...)")
+	command.Flags().Float32Var(&config.clientQPS, "client-qps", config.clientQPS, "server's client qps")
```

Comment:
maximum number of requests per second the server uses when communicating with Kubernetes when backing up and restoring

Would that be okay?

Reply:
sgtm!

Reply:
Wanted to jot this down before I forget - I'm not sure if "maximum" is the best word here, since it's possible to burst above the qps. It's more like the average sustained requests rate.

Reply:
"quota" seems like one of the right words here. Something like "standard quota of requests per second to Kubernetes"?

Reply:
Okay, let me know the final wording for qps and burst param descriptions!